src/components/active-students/ActiveStudentsBlock.tsx

```diff
@@ -18,18 +17,27 @@ const ActiveStudentsBlock = () => {
   const { t } = useTranslation()
   const navigate = useNavigate()
 
-  const getMyCooperations = useCallback(
-    () => cooperationService.getCooperations({ limit: 3, status: 'active' }),
-    []
-  )
+  const fetchMyCooperations = async (): Promise<
+    ItemsWithCount<Cooperation>
+  > => {
+    const response = await cooperationService.getCooperations({
+      limit: 3,
+      status: 'active'
+    })
+    return response.data as ItemsWithCount<Cooperation>
+  }
```

Comment:
Please, use `baseSerivce` inside `getCooperation` function for correct typed function. You will not have to do `response.data` manually and cast it as `ItemsWithCount<Cooperation>`
```suggestion
  const fetchMyCooperations = () => {
    return cooperationService.getCooperations({
      limit: 3,
      status: 'active'
    })
  }
```